### PR TITLE
Module1:Modal popup

### PR DESCRIPTION
**Pull request by derrick1451**
- ### Modal popup
- In this module, we implemented the popup window on both the mobile and desktop screens 
- we achieved this by

1. Update the HTML and generated the work-section dynamically
2. Edit the CSS files

### CODE REVIEW

**Comment by AshakaE** [comment]
### Hi Team 🙋 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request ❌ 

You have submitted a project, that doesn't have commit history from every team member, as a team project everyone is expected to have an input in the project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by derrick1451** [comment]
my coding partner actually left the program so am really supposed to do the work individually

**Review by amejid — COMMENTED** [review]
Hi @derrick1451, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have commits from your coding partner. Please contact student success and provide a screenshot of the permission given by them to work solo.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by derrick1451** [comment]
[image: Screenshot from 2023-04-19 16-19-24.png]

**Comment by derrick1451** [comment]
thats the recent email l received from success permitting to work on the
task individually

**Review by franclobo — COMMENTED** [review]
Hi @derrick1451, 

terminator

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is a pair-programming activity and doesn't have commits from your partner. The image that proves that student success has authorized you to work alone has not charged correctly 👇🏽 

image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by derrick1451** [comment]
these are the screenshots to prove that student success has permitted me

**Comment by derrick1451** [comment]
Screenshot from 2023-04-19 16-44-54
Screenshot from 2023-04-19 16-44-05

File: style.css [hunk 1]
```diff
@@ -86,6 +89,39 @@ a {
...
+}
+
+.nav-links {
+  display: flex;
+  list-style: none;
+  align-items: center;
+  gap: 12px;
+  padding-left: 30px;
+}
+
```
Comment (emmiiorji) [inline style.css:120]: Kindly adjust the design on mobile version to be as close to the design as possible. Check the illustration below for more detail.

Your design:
image

Figma design:
image

File: style.css [hunk 2]
```diff
@@ -635,6 +720,75 @@ label {
...
+    padding: 1rem;
+    flex-direction: column;
+    height: auto;
+    border-radius: 16px;
+  }
+
+  .d-header {
+    display: flex;
+    justify-content: space-between;
+  }
```
Comment (emmiiorji) [inline style.css]: On the desktop version, kindly adjust the design to be as close to the design as possible. Check the illustration below for details:

Your design:
image

Figma design:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @derrick1451, 
# Status: Changes Required ♻️

Good job so far! Sorry for having to work alone and the hitches experienced.

There are some issues that you still need to work on to go to the next project but you are almost there!

<div align='center'>

</div>

## Good Highlights✔️

- Popup appears on click of "See Project" button
- The popup is able to close using the close button
- Good use of array to hold all projects


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AdnanOlarmmi — APPROVED** [review]
Hi @derrick1451,

This is Adnan Is-haq, your beloved code reviewer, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved office man

## Highlights
- Nice one fixing the required changes.

## Optional suggestions
_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**